docs/pages/Hosting/ProjectToSiteMapping.adoc

```diff
@@ -0,0 +1,88 @@
+
+. <<Introduction>>
+. <<Continuous Deployment>>
+. <<Environment Variables>>
+.. <<NITRO_PRESET>>
+.. <<MODE>>
+.. <<Other Variables>>
+
+== Introduction
+
+This page provides an overview over the various site configuration options netlify offers. 
+We will go over the environment variables needed to deploy the PWA template as the configuration is already covered by the https://docs.netlify.com/[netlify documentation].
+We will also go over the continuous deployment configuration as it will be needed for mapping the various deployment stages (dev/qa/prod) of your FirstSpirit Projects to Netlify sites.
+We recommend using at least two sites. 
+One for edit or preview mode and one for release mode (as defined by the <<MODE>> environment variable).
+You can model all of the stages using the netlify continuous deployment configurations in conjunction with setting the environment variables for each deployment context individually.
+image:NetlifySiteMapping.png[Overview over our recommended netlify deployment mapping architecture]
+The mapping relies on the deploy preview feature of netlify.
+It deploys a preview of the site whenever a pull request is made against the production branch or selected branches.
+If you add for example a staging branch to the list of branches to deploy as part of the site configuration you will get a preview deployment of the site whenever a pull request is made against the staging branch.
+So in order to set up your dev stage you can configure your deploy previews to use data from the development environment.
+If you add staging specific environment variables to the staging branch deployment context you've got your staging stage covered.
+And finally if you add production specific environment variables to the production branch deployment context you've got your production stage set up.
+
+[WARNING]
+====
+Keep in mind that a pull request from staging to production will trigger a deploy preview using a development environment. 
+Unfortunately there is no other way to set this up as netlify only allows one set of variables for the deploy preview deployment context.
+====
+
+== Continuous Deployment
+
+We recommend setting the following properties here
+- Production branch: This property states which branch should be used to deploy the site. 
+This should be set to your main branch on github. 
+Or whichever branch contains your production level code.
+- Branch deploys: Allows to define whether any branches on github should also deployed as part of this site. 
+You can add individual branches to be deployed (eg. staging). 
+We would recommend adding your staging or qa branch to the list of individual branches.
+- Deploy previews: A preview is deployed when a pull request is made against the production branch 
+and redeployed whenever a new commit gets pushed onto the branch. 
+Deploy previews are not only useful because they let you test your current development branch in the netlify environment.
```

Comment:
Either you drop the "not only" here or you add something like "also" to the following sentence ;)